src/api/entities/confidential/ConfidentialAsset/types.ts

```diff
@@ -0,0 +1,10 @@
+import BigNumber from 'bignumber.js';
+
+import { Identity } from '~/internal';
+
+export interface ConfidentialAssetDetails {
+  ticker?: string;
```

Comment:
why is ticker not always defined?

I think it would be a good place to comment why if its intended